package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@divriots/dockit-react",
-  "version": "0.0.63",
```

Comment:
what is this? 😱  I didn't change it

Reply:
this is super strange, looks like the last master commit wasn't there when I branched from it, but it could not be true
anyhow, I fixed and it's no problem anymore